### PR TITLE
[dep. on #480] Refactoring stories for new docs concept

### PR DESCRIPTION
**Pull request by aniamusial**
# Related issue
#480 

# Scope of work
Added missing md files for docs.

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# component-description
+Component with add to cart button and quantity selector
```
Comment (EduardDopler) [inline packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.md]:
```suggestion
Add-to-cart [button](Button.html) and quantity input field with maximum stock validation.
```

File: packages/vue/src/components/molecules/SfAlert/SfAlert.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# component-description
+Alert message with icon and text
```
Comment (EduardDopler) [inline packages/vue/src/components/molecules/SfAlert/SfAlert.md]:
```suggestion
Component for displaying brief information of varying importance to the user, enriched with an icon.
```

File: packages/vue/src/components/molecules/SfBanner/SfBanner.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# component-description
+Banner with background image, title, subtitle and description
```
Comment (EduardDopler) [inline packages/vue/src/components/molecules/SfBanner/SfBanner.md]:
```suggestion
Banner component which features various text levels, a background and a [button](Button.html).
```

**Comment by EduardDopler** [comment]
@aniamusial Wow I love seeing this auto-generation of docs topic being worked on on so many ends 👏 thank you already!

If you want to, you can track your progress in issue #420. It's easier to see what parts are missing.

I've just suggested 3 changes for the components I had finished locally but haven't pushed yet because the auto-generation PR took off.

**Comment by przspa** [comment]
@aniamusial @EduardDopler what we should do on this PR? If we should merge it then @aniamusial, please resolve conflicts

**Comment by EduardDopler** [comment]
@pspaczek This PR depends on #480 (auto-generation of common usage). Once that PR is merged to develop, we can update this branch here (merge develop into it) and merge it too.

**Pull request merged**